tools/fp20compiler/nvparse_externs.h

```diff
@@ -1,6 +1,7 @@
 #ifndef NVPARSE_EXTERNS_H
 #define NVPARSE_EXTERNS_H
 
+#include "nvparse_errors.h"

```

Comment:
"fp20compiler: Include nvparse_errors.h for nvparse_errors type"

This commit title is bad. It assumes knowledge of where it does the change or why the type is necessary.

Reply:
It's used in the next line. It wasn't an issue before, because other users always include "nvparse_errors.h" first.
This PR adds code which doesn't include "nvparse_errors.h".

So either..

A. I should ditch this commit and always include "nvparse_errors.h".
B. Remove other manual inclusion of "nvparse_errors.h" and only include "nvparse_externs.h".